src/main/java/org/jenkins/tools/test/hook/WorkflowCpsHook.java

```diff
@@ -13,7 +14,7 @@ protected String getParentFolder() {
 
     @Override
     protected String getParentProjectName() {
-        return "workflow-cps-parent";
+        return "workflow-cps";
```

Comment:
Consistent with the tagging scheme of this repository, not that it matters since this was a fallback code path. Since this file was added recently, the mistake here just serves to illustrate how poor the old Javadoc was (corrected in this PR).